report/adult_income_predictor_report.qmd

```diff
@@ -36,6 +36,31 @@ To ensure that the analysis is not influenced by erroneous data, the inputs to t
 
 # EDA Analysis
 ## Data Summary
+In @tbl-despriction, it describes the number of unique categories and the missing values for the relevant categorical variables.
```

Comment:
Typo in ‘despriction’

Reply:
Also, like Daniel said in class last week. I think we don’t need to show the data to the reader as it is not important / relevant for them to know this in the analysis

Reply:
I just noticed that the milestone3 requires including a table to be rendered. But if you think it isn't needed, then I will close the pull request.